task/generate-odcs-compose/0.1/generate-odcs-compose.yaml

```diff
@@ -3,9 +3,12 @@ apiVersion: tekton.dev/v1beta1
 kind: Task
 metadata:
   name: generate-odcs-compose
-  build.appstudio.redhat.com/expires-on: "2025-03-15T00:00:00Z"
-  build.appstudio.redhat.com/expiry-message: 'This task is deprecated.
-    Please remove it from your pipeline.'
+  annotations:
+    build.appstudio.redhat.com/expires-on: "2025-03-15T00:00:00Z"
+    build.appstudio.redhat.com/expiry-message: 'This task is deprecated.
```

Comment:
This is not consistently applied on all expired tasks... Not asking you to fix, necessarily, just noticing.

Reply:
Just thinking about `build-and-push.sh` should skip building expired tasks. Does this make sense?